src/cache/local-cache.manager.ts

```diff
@@ -0,0 +1,25 @@
+import { ICacheManager, SetOptions } from "./cache.manager.interface";
```

Comment:
Can we maybe add tests for the two cache managers? (local and redis)

Reply:
sure